pkg/tfgen/generate_nodejs.go

```diff
@@ -732,9 +732,15 @@ func (g *nodeJSGenerator) emitNPMPackageMetadata(pack *pkg) error {
 	}
 	defer contract.IgnoreClose(w)
 
+	var packageName = g.info.JavaScript.PackageName
```

Comment:
NIT: It would be a little more idiomatic to write this as `packageName := g.info.JavaScript.PackageName`